lib/tasks/setup.rake

```diff
@@ -22,8 +22,16 @@ namespace :setup do
     missing_files = SchemaFileLoader.get_missing_downloads(vendor_dir)
     if missing_files.present?
       message = <<~MESSAGE
-        Download the following files from https://drive.google.com/drive/u/0/folders/1ssEXuz5WDrlr9Ng7Ukp6duSksNJtRATa
-        #{missing_files.map { |(filename, download_folder)| "#{filename} to vendor/#{download_folder}" }.join("\n")}
+                You need to manually download the following files from https://drive.google.com/drive/u/0/folders/1ssEXuz5WDrlr9Ng7Ukp6duSksNJtRATa
+                #{
+                  missing_files.map do |filename|
+                    matching_file = SchemaFileLoader::EFILE_SCHEMAS_FILENAMES.find do |(file_name, _dir)|
+                      File.basename(filename) == file_name
+                    end
+                    "#{filename} to vendor/#{matching_file[1]}"
+                  end.join("\n")
+                }
+        end
```

Comment:
the printout was telling me to download `[filename] to /vendor` only instead of into a specified folder within `/vendor` (i.e. `us_states`/`irs`)

Reply:
wait, i'm confused by this actually. it looks like the code as it was before your change should have been outputting vendor/irs or vendor/us_state depending on the values in missing_files. did you happen to see why `download_folder` was nil or empty, if that was indeed the case?